app/controllers/hq/partners_controller.rb

```diff
@@ -1,71 +1,44 @@
 class Hq::PartnersController < ApplicationController
-  before_action :authenticate_admin_user!
+  before_action :authenticate_admin_user!,
+                :new_partner_facade
   layout 'application'
 
   def index
-    @partners= Partner.all
+    @partners = Partner.all
   end
```

Comment:
You mean like:   `@partners_facade.load_partners :all` and `@partners_facade.load_partners :active`? This is great!